src/inmanta/protocol/methods.py

```diff
@@ -486,7 +488,10 @@ def get_version(tid: uuid.UUID, id: int, include_logs: bool = None, log_filter:
     :param id: The id of the version to retrieve
     :param include_logs: If true, a log of all operations on all resources is included
     :param log_filter: Filter log to only include actions of the specified type
-    :param limit: The maximal number of actions to return per resource (starting from the latest)
+    :param limit: The maximal number of actions to return per resource (starting from the latest),
+                    up to a maximum of 1000.
+                    To retrieve more entries, use /api/v2/resource_actions
+                    (:func:`~inmanta.protocol.methods_v2.get_resource_actions`)
```

Comment:
This refers to `resource_actions`, same as the method above. Is this a copy-paste error or is it just a more generic method than the ones in v1?

Reply:
the limit here applies to the log (also called resource actions, perhaps that should be cleaned up). 
There is no method in v1 that allows getting the next page. This v2 method is the only option.